Runtime/GameLiftConfiguration.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-namespace AmazonGameLift.Runtime
+namespace AmazonGameLiftPlugin.Core
 {
     [System.Serializable]
     public struct GameLiftConfiguration
     {
         public string AwsRegion;
         public string UserPoolClientId;
         public string ApiGatewayEndpoint;
```

Comment:
I think as we've discussed in today's meeting, this file solely contains settings used by the client to connect to the backend service for player auth? 

For server settings, we should store those separately. Which means the IsGameLiftAnywhere flag should live there?

Reply:
The IsGameLiftAnywhere flag is used similarly to how local testing worked before. if `(_configuration.ApiGatewayEndpoint.StartsWith("http://localhost"))` It is for the clients benefit not the server, the client will need to know whether to connect to the GameLiftAnywhere server or the deployed version.

Reply:
Where is the property populated? `GameLiftClientSettings` doesn't have `IsGameLiftAnywhere` and doesn't provide it when initializing `GameLiftConfiguration`